examples/deepcopy-gen/test/wholepkg/deepcopy_generated.go

```diff
@@ -54,6 +60,15 @@ func DeepCopy_wholepkg_ManualStruct(in interface{}, out interface{}, c *conversi
 	}
 }
 
+func DeepCopy_wholepkg_ManualStruct_Alias(in interface{}, out interface{}, c *conversion.Cloner) error {
+	{
```

Comment:
This is needed to do the `in := in.(*cast)` stuff.  without the extra nesting, it doesn't compile.  It has been there for a long time.